Code/GraphMol/CIPLabeler/CIPLabeler.h

```diff
@@ -20,6 +20,24 @@ class ROMol;
 
 namespace CIPLabeler {
 
+RDKIT_CIPLABELER_EXPORT bool decrementRemainingCallCountAndCheck();
+
+/*
+  Some very symmetrical mols can cause pseudo infinite processing
+  (e.g. dodecahedrane)
+  To avoid this a maxinum number of iterations can be set by the caller as a
+  parameter to assignCIPLabels.
+  If that maximum value is exceeded, the following error is thrown
+*/
+
+class MaxIterationsExceeded : public std::runtime_error {
```

Comment:
I believe that you want an export on this as well so that the exception is exposed across DLL boundaries
```suggestion
class RDKIT_CIPLABELER_EXPORT MaxIterationsExceeded : public std::runtime_error {
```